20_1 Button: add getting started (#1732)
DevExpress/devextreme-documentation: concepts/05 UI Components/Button/00 Getting started with Button/08 Stylize the Button.md

> **Specify a styling mode**           
> The styling mode defines whether a Button has a colored border (*"outlined"*), a colored background (*"contained*" - default), or none of them (*"text"*). We will use a colored border. For this, set the [stylingMode](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#stylingMode) property to *"outlined"*.
> 
> **Specify a type**                  
> The button [type](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type) property defines its color. The following types are available: *"danger"* (red), *"default"* (blue), *"normal"* (transparent), *"success"* (green), and *"back"* (transparent). The type of *"back"* is special - it ignores the **text**, **stylingMode** and other properties and transforms the UI component into a left arrow button. In this tutorial, we will set the **type** to *"success"*.

Reviewer: Since an illustration is worth a thousand words, let's just send the reader to the demo.

```suggestion
The Button supports several predefined color schemes. To specify one of them, set the [type](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type) property. You can also use the [stylingMode](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#stylingMode) property to customize the fill and borders. Refer to the [Predefined Types](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) demo for more information about these properties.
```

The Button supports several predefined color schemes. To specify one of them, set the [type](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type) property. You can also use the [stylingMode](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#stylingMode) property to customize the fill and borders. Refer to the [Predefined Types](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) demo for more information about these properties.

---
##### jQuery

    <!-- tab: index.js -->
    $(function() {
        $("#button").dxButton({
            // ...
            type: "success",
            stylingMode: "outlined"
        });
    });

##### Angular

    <!-- tab: app.component.html -->

>     <dx-button
>         <!-- ... -->

Reviewer: ```suggestion
    <dx-button ...
```

        stylingMode="outlined"
        type="success">
    </dx-button>

##### Vue

    <!-- tab: App.vue -->
    <template>

>         <DxButton
>             <!-- ... -->

Reviewer: ```suggestion
        <DxButton ...
```

            styling-mode="outlined"
            type="success"
        />
    </template>

##### React

    <!-- tab: App.js -->
    // ...

    function App() {
        return (
            <div className="App">

>                 <Button
>                     // ...

Reviewer: ```suggestion
                <Button ...
```
